wled00/FX_fcn.cpp

```diff
@@ -1326,10 +1338,10 @@ void WS2812FX::service() {
     if (!seg.isActive()) continue;
 
     // last condition ensures all solid segments are updated at the same time
-    if (nowUp > seg.next_time || _triggered || (doShow && seg.mode == FX_MODE_STATIC))
+    if (nowUp >= seg.next_time || _triggered || (doShow && seg.mode == FX_MODE_STATIC))
     {
       doShow = true;
-      unsigned delay = FRAMETIME;
+      unsigned frameDelay = FRAMETIME;
```

Comment:
Changing variable name just makes change harder to follow.

Reply:
This was intentionally - the name "delay" is a function, and I found it's better to avoid having a local with the same name.

Reply:
Avoiding name collisions is a good idea!  I wish it was separate commit, though - it makes it easier on us reviewers and you can put the rationale in the commit message so we know what it's for.

Reply:
Yeah, I'll make a separate commit in 0_15 only for the renaming. Then it should automatically  disappear from changes listed in this PR .

Reply:
done.